wolfcrypt/src/ecc.c

```diff
@@ -6217,6 +6238,9 @@ int wc_ecc_verify_hash_ex(mp_int *r, mp_int *s, const byte* hash,
    CRYS_ECDSA_VerifyUserContext_t sigCtxTemp;
    word32 msgLenInBytes = hashlen;
    CRYS_ECPKI_HASH_OpMode_t hash_mode;
+#elif defined(WOLFSSL_SILABS_SE_ACCEL)
+   // TODO:
```

Comment:
What is the TODO here? Can we clean this up? Recommend using `MAX_ECC_BYTES * 2` or maybe `ECC_MAX_CRYPTO_HW_SIZE * 2`.

Reply:
Done